R/rmarkdown.R

```diff
@@ -24,6 +24,9 @@ render_rmarkdown <- function(pkg, input, output, ..., copy_images = TRUE, quiet
     quiet = quiet
   )
 
+  on.exit(Sys.setenv(BSTINPUTS = Sys.getenv("BSTINPUTS")), add = TRUE)
```

Comment:
Please use the `env` argument to `r_safe`

Reply:
Ok, I didn't know this feature.